app/controllers/catalog_controller.rb

```diff
@@ -40,6 +40,7 @@ class CatalogController < ApplicationController
 
     config.index.document_component = Earthworks::SearchResultComponent
     config.index.title_field = Settings.FIELDS.TITLE
+    config.index.search_bar_component = Earthworks::SearchBarComponent
```

Comment:
Do we want to use this component for the show view too?

Reply:
I don't see a config.show.search_header_component in blacklight and it looks like the show page is using the custom search_header_component